### PR TITLE
Update shadow-on-ubuntu

### PR DESCRIPTION
**Pull request by kewde**
### Changelog
- Added the logo to README.MD
### Description of the change

Added the logo!

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@kewde__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

**Comment by kewde** [comment]
I wonder, how  or when does a template get added to the website?
https://azure.microsoft.com/en-us/documentation/templates/

**Comment by kewde** [comment]
Why is this PR still not accepted?

**Comment by singhkay** [comment]
@dasource As the author, can you review this change?

File: shadow-on-ubuntu/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# Shadow Public Blockchain Node
```
Comment (singhkay) [inline shadow-on-ubuntu/README.md]: Please leave the heading

**Comment by gatneil** [comment]
Sorry for the delay; do we still want to merge this in? If so, the first step is to resolve the merge conflicts :)

**Comment by gatneil** [comment]
No response; closing

**Pull request closed without merging**